src/main/scala/tech/cryptonomic/conseil/routes/openapi/TupleFlattenHelper.scala

```diff
@@ -0,0 +1,39 @@
+package tech.cryptonomic.conseil.routes.openapi
+
+trait TupleFlattenHelper {
```

Comment:
I will find the original gist to refer to, so we know where everything came from.

Reply:
it's adapted from `travisbrown`'s solution.
My fork is here: https://gist.github.com/ivanopagano/282ec7112e29e63e603491961bdb2a5c

Reply:
added link to the `travisbrown`'s original solution in the comment